src/m.ts

```diff
@@ -0,0 +1,1294 @@
+          name: "unmute",
+          description: "Unmute iTunes",
+        },
+        {
+          name: "vol",
+          description: "Volume",
+          args: {
+            name: "up/down/#",
+            description: "Up/Down/Number",
+            suggestions: ["up", "down" /*, "#",*/],
+            isOptional: true,
+          },
+        },
+        {
+          name: "stop",
+          description: "Stop track",
+        },
+        {
+          name: "quit",
+          description: "Quit iTunes",
+        },
+      ],
+    },
+    {
+      name: "localhost",
+      description: "Manage localhost",
+      subcommands: [
+        {
+          name: "ls",
+          description: "List current records in localhost",
+        },
+        {
+          name: "add",
+          description: "Add a new host to the localhost file",
+          args: [
+            {
+              name: "IP",
+              description: "IP of the host",
+            },
+            {
+              name: "HOSTNAME",
+              description: "Hostname of the host",
+            },
+          ],
+        },
+        {
+          name: "remove",
+          description: "Remove a host from the localhost file",
+          args: {
+            name: "HOSTNAME",
+            description: "Hostname of the host",
+          },
+        },
+      ],
+    },
+    {
+      name: "lock",
+      description: "Lock session",
+    },
+    {
+      name: "network",
+      description: "Manage network",
+      subcommands: [
+        {
+          name: "ls",
+          description: "List network interfaces",
+        },
+        {
+          name: "location",
+          description: "Manage location",
+          subcommands: [
+            {
+              name: "ls",
+              description: "List locations",
+            },
+            {
+              name: "create",
+              description: "Create a location",
+              args: {
+                name: "name",
+                description: "Name of the location",
+              },
+            },
+            {
+              name: "delete",
+              description: "Delete a location",
+              args: {
+                name: "name",
+                description: "Name of the location",
+              },
+            },
+            {
+              name: "switch",
+              description: "Switch location",
+              args: {
+                name: "name",
+                description: "Name of the location",
+              },
+            },
+          ],
+        },
+      ],
+    },
+    {
+      name: "nosleep",
+      description: "Manage no sleep",
+      subcommands: [
+        {
+          name: "until",
+          description: "No sleep until",
+          subcommands: [
+            {
+              name: "pid",
+              description: "No sleep until process id ends",
+              args: {
+                name: "pid",
+                description: "Process id",
+              },
```

Comment:
Can reuse the generator from `kill.ts`:
```ts
function getPidIcon(path: string): string {
  const idx = path.indexOf(".app/");
  if (idx === -1) {
    return "fig://icon?type=gear";
  }
  return "fig://" + path.slice(0, idx + 4);
}

const generatePids: Fig.Generator = {
  script: "ps axo pid,comm | sed 1d",
  postProcess: (result) => {
    return result.split("\n").map((line) => {
      const [pid, path] = line.trim().split(/\s+/);
      const name = path.slice(path.lastIndexOf("/") + 1);
      return {
        name: pid,
        description: path,
        displayName: `${pid} (${name})`,
        icon: getPidIcon(path),
      };
    });
  },
};
```

```suggestion
              args: {
                name: "pid",
                description: "Process id",
                generators: generatePids,
              },
```